plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/HiveS3Config.java

```diff
@@ -68,6 +68,7 @@
     private boolean requesterPaysEnabled;
     private boolean s3StreamingUploadEnabled = true;
     private DataSize s3StreamingPartSize = DataSize.of(16, MEGABYTE);
+    private String s3RoleSessionName = "trino-session";
```

Comment:
Do you need this config once you are using s3 mapping?

Reply:
No, you are right. Removed in latest update.